container/container_windows.go

```diff
@@ -38,19 +42,32 @@ func (container *Container) CreateDaemonEnvironment(_ bool, linkedEnv []string)
 func (container *Container) UnmountIpcMounts(unmount func(pth string) error) {
 }
 
+// SecretMountPath returns the path of the secret mount for the container
+func (container *Container) SecretMountPath() string {
+	return filepath.Join(container.Root, "secrets")
+}
+
 // IpcMounts returns the list of Ipc related mounts.
 func (container *Container) IpcMounts() []Mount {
 	return nil
 }
 
 // SecretMount returns the mount for the secret path
 func (container *Container) SecretMount() *Mount {
```

Comment:
True, there are at least a couple of these simple functions that could be shared. It might make sense to move them to `container.go` but I think I'd like to keep them separate for now.